node/hyperbahn/service_proxy.js

```diff
@@ -84,6 +92,26 @@ function handleRequest(req, buildRes) {
         return;
     }
 
+    // apply rate limiter
+    if (req.serviceName !== 'hyperbahn') {
```

Comment:
Rate limiting itself should be behind a flag; we should also have a method for toggling it at run time.

We need to be able to write an admin script to control this.